config/boards/radxa-zero-3w.csc

```diff
@@ -0,0 +1,50 @@
+# Rockchip RK3566 quad core
+BOARD_NAME="Radxa ZERO 3W"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER=""
+BOOTCONFIG="radxa-zero3-rk3566_defconfig"
+KERNEL_TARGET="vendor,current,edge"
+KERNEL_TEST_TARGET="vendor"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3566-radxa-zero-3w.dtb"
+IMAGE_PARTITION_TABLE="gpt"
+BOOT_SCENARIO="spl-blobs"
+BOOTFS_TYPE="fat" # Only for vendor/legacy
+AIC8800_TYPE="sdio"
+enable_extension "radxa-aic8800"
+
+function post_family_config_branch_edge__use_mainline_dtb_name() {
+	unset BOOT_FDT_FILE # boot.scr will use whatever u-boot detects and sets 'fdtfile' to
+	unset BOOTFS_TYPE   # mainline u-boot can boot ext4 directly
+}
+
+# Override family config for this board; let's avoid conditionals in family config.
+function post_family_config__radxa-zero3_use_vendor_uboot() {
```

Comment:
This will force `current` and `edge` to use old vendor uboot as well. Is this intended?

Reply:
I copied this part straight from radxa-zero3.csc, and didn't touch it, so I am unsure of the history here.